pilot/pkg/config/kube/ingress/status.go

```diff
@@ -23,9 +23,9 @@ import (
 	"strings"
 	"time"
 
-	v1 "k8s.io/api/core/v1"
+	coreV1 "k8s.io/api/core/v1"
 	"k8s.io/api/extensions/v1beta1"
-	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
We've been using meta_v1 and similar pretty consistently in the codebase - maybe this is not the proper PR to change this in few files.

Reply:
It's a mix ... goland complains about using snakecase.